app/views/smart_answers/question.html.erb

```diff
@@ -39,4 +39,10 @@
 
     <%= render 'smart_answers/previous_answers' %>
   </div>
+
+  <div class="govuk-grid-column-one-third govuk-!-margin-top-9">
+    <% if @presenter.show_escape_link? %>
+      <%= render "smart_answers/escape-link" %>
+    <% end %>
+  </div>
```

Comment:
This doesn't quite look right on Safari - the link appears at the top of the page. Maybe putting it the layout, nested under https://github.com/alphagov/smart-answers/blob/a6d91ff8868a690d852e118cf32c415823595eb3/app/views/layouts/application.html.erb#L31

Reply:
Hi @theseanything - resolved - thanks 👍